src/resources/swagger.yaml

```diff
@@ -785,15 +793,77 @@ paths:
         '200':
           description: 'Donor with updated stats'
 
+  /submission/schema:
+    get:
+      deprecated: true
+      summary: 'use  `GET /dictionary` instead'
+      tags:
+        - Deprecated
+    patch:
+      deprecated: true
+      summary: 'use  `POST /dictionary/migration/run` instead'
+      tags:
+        - Deprecated
+  /submission/schema/dry-run-update:
+    post:
+      deprecated: true
+      summary: 'use `POST /dictionary/migration/dry-run-update` instead'
+      tags:
+        - Deprecated
+  /submission/schema/migration/resume:
+    post:
+      deprecated: true
+      summary: 'use `POST /dictionary/migration/resume` instead'
+      tags:
+        - Deprecated
+  /submission/schema/migration/{migrationId}:
+    get:
+      deprecated: true
+      summary: 'use `GET /dictionary/migration/{migrationId}` instead'
+      tags:
+        - Deprecated
+  /submission/schema/migration/:
+    get:
+      deprecated: true
+      summary: 'use `GET /dictionary/migration` instead'
+      tags:
+        - Deprecated
+  /submission/schema/changes:
+    get:
+      deprecated: true
+      summary: 'use `GET /dictionary/changes` instead'
+      tags:
+        - Deprecated
+  /submission/schema/list:
+    get:
+      deprecated: true
+      summary: 'use `GET /dictionary/list` instead'
+      tags:
+        - Deprecated
+  /submission/schema/template/all:
+    get:
+      deprecated: true
+      summary: 'use `GET /dictionary/template/all` instead'
+      tags:
+        - Deprecated
+  /submission/schema/template/{schemaName}:
+    get:
+      deprecated: true
+      summary: 'use `GET /dictionary/template/{schemaName}` instead'
+      tags:
+        - Deprecated
+
```

Comment:
^ Swagger-ui will render even if you don't specify the response body. I changed my mind, it is probly better to keep them for our own sanity so we can look up what is changed for what.

Reply:
and also we'll know what to remove when its time to remove them.

Reply:
sounds good not a big deal